jsk_spot_robot/jsk_spot.rosinstall

```diff
@@ -0,0 +1,83 @@
+#  - https://github.com/jsk-ros-pkg/jsk_recognition/pull/2596
+- git:
+    local-name: jsk-ros-pkg/jsk_recognition
+    uri: https://github.com/sktometometo/jsk_recognition.git
+    version: develop/spot
+# spot-jsk will use packages below.
+#  - dialogflow-executive
+#  - webrtcvad_ros
+#  - switchbot_ros
+#  - respeaker_ros
+#  - ros_speech_recognition
+#  - aques_talk
+# Until PRs below are merged, we need to use develop/spot branch
+#  - webrtcvad_ros
+#    + https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/282
+#  - switchbot_ros update
+#    + https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/278
+#  - dialogflow_ros update
+#    + https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/242
+- git:
+    local-name: jsk-ros-pkg/jsk_3rdparty
+    uri: https://github.com/sktometometo/jsk_3rdparty.git
+    version: develop/spot
+# Until PR below is merged, we need to use develop/spot branch
+#  - https://github.com/jsk-ros-pkg/jsk_common/pull/1698
+- git:
+    local-name: jsk-ros-pkg/jsk_common
+    uri: https://github.com/sktometometo/jsk_common.git
+    version: develop/spot
+# Until PRs below are merged, we need to use develop/spot branch
+#  - person following demo
+#    + https://github.com/jsk-ros-pkg/jsk_demos/pull/1343
+#  - mech office demo
+#    + https://github.com/jsk-ros-pkg/jsk_demos/pull/1341
+#  - sing and dance demo
+#    + https://github.com/jsk-ros-pkg/jsk_demos/pull/1340
+#  - person leading demo
+#    + https://github.com/jsk-ros-pkg/jsk_demos/pull/1339
+#  - watchdog demo
+#    + https://github.com/jsk-ros-pkg/jsk_demos/pull/1332
+#    + patches
+#      - https://github.com/tongtybj/jsk_demos/pull/2
+#      - https://github.com/tongtybj/jsk_demos/pull/1
+- git:
+    local-name: jsk-ros-pkg/jsk_demos
+    uri: https://github.com/sktometometo/jsk_demos.git
+    version: develop/spot
+# spot-ros is required for spot
+# This is a develop branch for jsk version.
+# We need to use it until it is merged to master
+- git:
+    local-name: spot-ros
+    uri: https://github.com/sktometometo/spot_ros.git
+    version: develop/spot
+# app_manager and utilities are requied
+- git:
+    local-name: PR2/app_manager
+    uri: https://github.com/PR2/app_manager.git
+    version: kinetic-devel
+- git:
+    local-name: knorth55/app_manager_utils
+    uri: https://github.com/knorth55/app_manager_utils.git
+    version: master
+- git:
+    local-name: knorth55/audio_video_recorder
+    uri: https://github.com/knorth55/audio_video_recorder.git
+    version: main
+# fetch15 version of roswww is necessary for basic authentication
+- git:
+    local-name: tork-a/roswww
+    uri: https://github.com/knorth55/roswww.git
+    version: fetch15
+# visualization_rwt is not released for melodic
+- git:
+    local-name: tork-a/visualization_rwt
+    uri: https://github.com/tork-a/visualization_rwt.git
+    version: kinetic-devel
+# aerial_robot packages are requied for msgs for spot spinal
```

Comment:
Spot use spinal board of aerial_robot. which are defined in spinal.

Reply:
Only this package https://github.com/sktometometo/aerial_robot/tree/develop/spot/aerial_robot_nerve/spinal is required.

Reply:
humm...
@tongtybj put everything in one repository and try to remove external dependency. This seems efficient, but lacks  extensibility.
For example if @tongtybj uses https://github.com/rosflight/rosflight/blob/master/rosflight_msgs/msg/Barometer.msg (which is released as deb file https://github.com/ros/rosdistro/search?q=rosflight&type=commits), then we do not need to depend on this repository?  I know this is huge change so it is hard to switch them...

Reply:
Drone team shrink size of ros message as small as possible. So usual ROS message definition like Header seems not to fit to their needs.
But spot can use it instead because we dont use spinal board such way. ofcourse we have to re-write firmware of the board to use usual ROS messages like rosflight_msgs, sensor_msgs so on.